soc/st/stm32/stm32f7/Kconfig.soc

```diff
@@ -0,0 +1,64 @@
+# ST Microelectronics STM32F7 MCU line
+
+# Copyright (c) 2018 Yurii Hamann
+# Copyright (c) 2022, Rtone.
+# Copyright (c) 2023, Rahul Arasikere.
+# Copyright (c) 2023 Evan Perry Grove
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_STM32F7X
+	bool
+	select SOC_FAMILY_STM32
+
+config SOC_SERIES
+	default "stm32f7" if SOC_SERIES_STM32F0X
+
+config SOC_STM32F722XX
+	bool
+	select SOC_SERIES_STM32F7X
+
+config SOC_STM32F723XX
+	bool
+	select SOC_SERIES_STM32F7X
+
+config SOC_STM32F745XX
+	bool
+	select SOC_SERIES_STM32F7X
+
+config SOC_STM32F746XX
+	bool
+	select SOC_SERIES_STM32F7X
+
+config SOC_STM32F756XX
+	bool
+	select SOC_SERIES_STM32F7X
+
+config SOC_STM32F750XX
+	bool
+	select SOC_SERIES_STM32F7X
+
+config SOC_STM32F765XX
+	bool
+	select SOC_SERIES_STM32F7X
+	select CPU_HAS_FPU_DOUBLE_PRECISION
+
+config SOC_STM32F767XX
+	bool
+	select SOC_SERIES_STM32F7X
+	select CPU_HAS_FPU_DOUBLE_PRECISION
+
+config SOC_STM32F769XX
+	bool
+	select SOC_SERIES_STM32F7X
+	select CPU_HAS_FPU_DOUBLE_PRECISION
```

Comment:
`CPU_HAS_FPU_DOUBLE_PRECISION` flags go in `Kconfig`

Reply:
Right, I really need to get use to it. Thanks

Reply:
So, I actually moved those to soc/st/stm32/stm32f7/Kconfig.defconfig.stm32f7... files, which I think makes more sense.